tests/test_github.py

```diff
@@ -1,6 +1,7 @@
 # pylint: disable=no-member,invalid-name,redefined-outer-name
 import pytest
 import responses
+import typing
```

Comment:
Will fix this mypy complaint.

Reply:
that lint is weird, why does `typing` fail and not the other imports?